modules/logging-release-notes-5-8-4.adoc

```diff
@@ -0,0 +1,54 @@
+// module included in /logging/logging-5-8-release-notes
+:_mod-docs-content-type: REFERENCE
+[id="logging-release-notes-5-8-4_{context}"]
+= Logging 5.8.4
+This release includes link:https://access.redhat.com/errata/RHBA-2024:1065[OpenShift Logging Bug Fix Release 5.8.4].
+
+[id="logging-release-notes-5-8-4-bug-fixes"]
+== Bug fixes
+
+* Before this update, the developer console's logs did not account for the current namespace, resulting in query rejection for users without cluster-wide log access. With this update, all supported OCP versions ensure correct namespace inclusion. (link:https://issues.redhat.com/browse/LOG-4905[LOG-4905])
+
+* Before this update, the Cluster Logging Operator deployed `ClusterRoles` supporting LokiStack deployments only when the default log output was  LokiStack. With this update, the roles are split into two groups: read and write. The write roles deploys based on the setting of the default log storage, just like all the roles used to do before. The read roles deploys based on whether the logging console plugin is active. (link:https://issues.redhat.com/browse/LOG-4987[LOG-4987])
+
+* Before this update, multiple `ClusterLogForwarders` defining the same input receiver name had their service endlessly reconciled because of changing `ownerReferences` on one service. With this update, each receiver input will have its own service named with the convention of `<CLF.Name>-<input.Name>`. (link:https://issues.redhat.com/browse/LOG-5009[LOG-5009])
+
+* Before this update, the `ClusterLogForwarder` did not report errors when forwarding logs to cloudwatch without a secret. With this update, the following error message appears when forwarding logs to cloudwatch without a secret: `secret must be provided for cloudwatch output`. (link:https://issues.redhat.com/browse/LOG-5021[LOG-5021])
+
+* Before this update, the `log_forwarder_input_info` included `application`, `infrastructure`, and `audit` input metric points. With this update, `http` is also added as a metric point. (link:https://issues.redhat.com/browse/LOG-5043[LOG-5043])
```

Comment:
Is https already a metric point?

Reply:
As mentioned in the bug (https://issues.redhat.com/browse/LOG-5043) and PR (https://github.com/openshift/cluster-logging-operator/pull/2357) `http` is added as a metric point.